masterlist.yaml

```
@@ -831,6 +831,45 @@ plugins:
       - 'Order_Of_Sanctity.ESP'
       - 'Walkers of Morrowind.esp'
 
+   ###### Gameplay - Quests ######
+     ### Rise of House Telvanni ###
+  - name: 'Rise of House Telvanni.esm'
+    url:
+      - 'https://www.nexusmods.com/morrowind/mods/27545/'
+      - 'https://www.nexusmods.com/morrowind/mods/49231/'
```

This file isn't on the Beautiful Cities of Morrowind mod page.
```suggestion
    url: [ 'https://www.nexusmods.com/morrowind/mods/27545/' ]
```